routes/api.php

```diff
@@ -19,6 +19,8 @@
     Route::get('/countries')->controller('CountriesController')->action('index'),
     Route::get('/countries/{id}')->controller('CountriesController')->action('getById'),
     Route::get('/timezones')->controller('TimeZonesController'),
+    Route::crud('/countries/{countriesId}/states')->controller('StatesController'),
```

Comment:
@FredPeal lets have it all under 1 controller , lets use countries and each of this calls be a diff method